skyportal/handlers/api/thumbnail.py

```diff
@@ -183,20 +186,20 @@ def delete(self, thumbnail_id):
 
 
 def create_thumbnail(thumbnail_data, thumbnail_type, source_id, photometry_obj):
-    basedir = Path(os.path.dirname(__file__))/'..'/'..'
+    basedir = Path(os.path.dirname(__file__)) / '..' / '..'
     if os.path.abspath(basedir).endswith('skyportal/skyportal'):
-        basedir = basedir/'..'
+        basedir = basedir / '..'
     file_uri = os.path.abspath(
-        basedir/f'static/thumbnails/{source_id}_{thumbnail_type}.png')
+        basedir / f'static/thumbnails/{source_id}_{thumbnail_type}.png')
     if not os.path.exists(os.path.dirname(file_uri)):
-        (basedir/'static/thumbnails').mkdir(parents=True)
+        (basedir / 'static/thumbnails').mkdir(parents=True)
     file_bytes = base64.b64decode(thumbnail_data)
     im = Image.open(io.BytesIO(file_bytes))
     if im.format != 'PNG':
         raise ValueError('Invalid thumbnail image type. Only PNG are supported.')
-    if not (100, 100) <= im.size <= (500, 500):
+    if not all(100 <= x <= 500 for x in im.size):
         raise ValueError('Invalid thumbnail size. Only thumbnails '
-                        'between (100, 100) and (500, 500) allowed.')
+                         'between (100, 100) and (500, 500) allowed.')
```

Comment:
Is there a reason for the lower-limit on the thumbnail size? LSST will be providing 31x31 pixel images it seems (e.g. https://github.com/lsst-dm/sample-avro-alert/tree/master/examples). Could open a separate issue or perhaps we just lower it...?

Reply:
Sure - remove it altogether or just lower it to 16?

Reply:
Done.